### PR TITLE
Bxmsdoc 5232 master

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: doc-content/enterprise-only/openshift/secret-adminuser-create-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+endif::[]
+ifeval::["{context}"=="openshift-managed"]
+:INCLUDECENTRAL: {CENTRAL} Monitoring
+endif::[]
+//immutable handled separately, so do not define INCLUDECENTRAL for it
+
+
+= Creating the secret for the administrative user
+
+You must create a secret that contains the user name and password for a {PRODUCT} administrative user account. This secret is required for deploying {PRODUCT} using any template except the trial template.
```
Comment (ruromero) [inline doc-content/enterprise-only/openshift/secret-adminuser-create-proc.adoc]:
```suggestion
You must create a generic secret that contains the user name and password for a {PRODUCT} administrative user account. This secret is required for deploying {PRODUCT} using any template except the trial template.
```
Maybe it should be mentioned that the secret keys must be KIE_ADMIN_USER and KIE_ADMIN_PWD

File: doc-content/enterprise-only/openshift/secret-adminuser-create-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+= Creating the secret for the administrative user
+
+You must create a secret that contains the user name and password for a {PRODUCT} administrative user account. This secret is required for deploying {PRODUCT} using any template except the trial template. 
+
+If you are using multiple templates to deploy components of {PRODUCT}, use the same secret for all these deployments. The components utilize this user account to communicate with each other.
+
+ifdef::INCLUDECENTRAL[]
+You can also use this user account to log in to {INCLUDECENTRAL}. If you do not use RH-SSO or LDAP authentication, use this account to create other user accounts.
```
Comment (ruromero) [inline doc-content/enterprise-only/openshift/secret-adminuser-create-proc.adoc]: I don't understand this sentence.
If you do not use RH-SSO or LDAP, the user won't be able to create any more user accounts.
If you do use RH-SSO or LDAP, these credentials will be used by KIE Server and Business/Decision Central to communicate with each other and must exist in RH-SSO or LDAP with the same exact values as it is mentioned in the next sections.

File: doc-content/enterprise-only/openshift/secret-adminuser-create-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+endif::INCLUDECENTRAL[]
+ifdef::PAM[]
+ifeval::["{context}"=="openshift-immutable"]
+If you deploy the immutable monitoring template, you can also use this user account to log in to {CENTRAL} Monitoring. If you do not use RH-SSO or LDAP authentication, use this account to create other user accounts.
```
Comment (ruromero) [inline doc-content/enterprise-only/openshift/secret-adminuser-create-proc.adoc]: Same comment as before

File: doc-content/enterprise-only/openshift/secret-adminuser-create-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+[IMPORTANT]
+====
+If you use RH-SSO or LDAP authentication, the same user with the same password must be configured in your authentication system with the `kie-server,rest-all,admin` roles for {PRODUCT}.
+====
+
+.Procedure
+
+// the name is rhpam-credentials even for RHDM - this is the example value used in templates
+Use the `oc` command to generate a secret named `kie-admin-user-secret` from the user name and password:
```
Comment (ruromero) [inline doc-content/enterprise-only/openshift/secret-adminuser-create-proc.adoc]:
```suggestion
Use the `oc` command to generate a generic secret named `rhpam-credentials` from the user name and password:
```
Maybe mention the literal keys must be KIE_ADMIN_USER and KIE_ADMIN_PWD?

File: doc-content/enterprise-only/openshift/secret-adminuser-create-proc.adoc [hunk 5]
```diff
@@ -17,16 +17,18 @@ endif::[]
 
 = Creating the secret for the administrative user
 
-You must create a secret that contains the user name and password for a {PRODUCT} administrative user account. This secret is required for deploying {PRODUCT} using any template except the trial template. 
+You must create a generic secret that contains the user name and password for a {PRODUCT} administrative user account. This secret is required for deploying {PRODUCT} using any template except the trial template. 
+
+The secret must contain the user name and password as literals. key name for the user name is `KIE_ADMIN_USER`. The key name for the password is `KIE_ADMIN_PWD`.
```
Comment (ruromero) [inline doc-content/enterprise-only/openshift/secret-adminuser-create-proc.adoc]:
```suggestion
The secret must contain the user name and password as literals. The key name for the user name is `KIE_ADMIN_USER`. The key name for the password is `KIE_ADMIN_PWD`.
```

File: doc-content/enterprise-only/openshift/additional-server-managed-deploy-assy.adoc [hunk 6]
```diff
@@ -9,6 +9,8 @@ ifdef::context[:parent-context: {context}]
 
 You can deploy an additional managed {KIE_SERVER} to an authoring or managed environment.  Deploy the server in the same project as the {LOCAL_CENTRAL} deployment.
 
+In order for the {LOCAL_CENTRAL} to connect to the {KIE_SERVER}, you must enable the `OpenShiftStartupStrategy` setting for your environment, as described in <<startupstrategy-enable-proc>>.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/openshift/additional-server-managed-deploy-assy.adoc]: Suggest the following minor rewrite:

You must enable the `OpenShiftStartupStrategy` setting for your environment for the {LOCAL_CENTRAL} to connect to the {KIE_SERVER} as described in <<startupstrategy-enable-proc>>.

File: doc-content/enterprise-only/openshift/startupstrategy-enable-proc.adoc [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+In an environment deployed using {PRODUCT} authoring templates, {CENTRAL} manages one {KIE_SERVER}.  
+ifdef::PAM[]
+If you use the high-avalability authoring template or if you modified the single authoring template to use a database server other than an embedded H2 database, you
+endif::PAM[]
+ifdef::DM[]
+You
+endif::DM[]
+can scale the {KIE_SERVER} pod, but all the copies execute the same services.
+
+In order to connect additional {KIE_SERVERS} to {CENTRAL}, you must enable the `OpenShiftStartupStrategy` setting in the environment. With this setting, {CENTRAL} automatically discovers {KIE_SERVERS} in the same namespace and these {KIE_SERVERS} can be configured to connect to the {CENTRAL}.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/openshift/startupstrategy-enable-proc.adoc]: Suggest the following minor rewrite:

You must enable the `OpenShiftStartupStrategy` setting in the environment to connect additional {KIE_SERVERS} to {CENTRAL}. With this setting, {CENTRAL} automatically discovers {KIE_SERVERS} in the same namespace and these {KIE_SERVERS} can be configured to connect to the {CENTRAL}.

File: doc-content/enterprise-only/openshift/template-deploy-rhsso-proc.adoc [hunk 8]
```diff
@@ -69,24 +63,14 @@ Do not configure LDAP authentication and RH-SSO authentication in the same deplo
...
-The following users are required in order to set the parameters for the environment:
-** An administrative user with the `kie-server,rest-all,admin` roles. This user can administer and use the environment. {KIE_SERVERS} use this user to authenticate with {LOCAL_CENTRAL}.
-** A server user with the `kie-server,rest-all,user` roles. This user can make REST API calls to the {KIE_SERVER}. {LOCAL_CENTRAL} uses this user to authenticate with {KIE_SERVERS}.
-endif::sso_serveruser[]
-ifndef::sso_serveruser[]
-In order to set the parameters for the environment, an administrative user with the `kie-server,rest-all,admin` roles is required. The default user name for this user is `adminUser`. This user can administer and use the environment.
-endif::sso_serveruser[]
+* User names and passwords for {PRODUCT} are created in the RH-SSO authentication system. For a list of the available roles, see <<roles-users-con>>. 
++
+As a minimum, you must create a user with the username and password configured in the secret for the administrative user, as described in <<secret-adminuser-create-proc>>. This user must have the `kie-server,rest-all,admin` roles.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/openshift/template-deploy-rhsso-proc.adoc]: Minor rewrite suggestion:

You must create a user with the username and password configured in the secret **<MH> "secret" what? Is this a property?</MH>** for the administrative user, as described in <<secret-adminuser-create-proc>>. This user must have the `kie-server,rest-all,admin` roles.

File: doc-content/enterprise-only/openshift/environment-authoring-con.adoc [hunk 9]
```diff
@@ -14,16 +15,27 @@ If necessary, you can create separate environments, so that you can use one depl
 For {PRODUCT}, the procedures to deploy an authoring environment and a managed server environment are the same. You must first deploy an authoring environment template, consisting of {CENTRAL} and one {KIE_SERVER}. 
 
 If necessary, you can deploy additional {KIE_SERVER} templates in the same namespace to create an environment with multiple {KIE_SERVERS}. This environment can be a managed server environment for staging and production deployment of services. 
+
 endif::DM[]
+To enable {CENTRAL} to connect to additional {KIE_SERVERS}, you need to enable the `OpenShiftStartupStrategy` setting. To enable this seting, you must complete an additional manual step after deploying the authoring environment template.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/openshift/environment-authoring-con.adoc]: To enable {CENTRAL} to connect to additional {KIE_SERVERS}, enable the `OpenShiftStartupStrategy` setting. To enable this setting, you must complete an additional manual step after deploying the authoring environment template.

File: doc-content/enterprise-only/openshift/environment-authoring-single-modify-proc.adoc [hunk 10]
```diff
@@ -3,6 +3,7 @@
 
 By default, the single authoring template uses the H2 database with permanent storage. If you prefer to create a MySQL or PostgreSQL pod or to use an external database server (outside the OpenShift project), you need to modify the template before deploying the environment.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/openshift/environment-authoring-single-modify-proc.adoc]: I would remove "you need to":
By default, the single authoring template uses the H2 database with permanent storage. If you prefer to create a MySQL or PostgreSQL pod or to use an external database server (outside the OpenShift project), modify the template before deploying the environment.

File: doc-content/enterprise-only/openshift/environment-authoring-con.adoc [hunk 11]
```diff
@@ -14,16 +15,27 @@ If necessary, you can create separate environments, so that you can use one depl
...
 
 Depending on your needs, you can deploy either a single authoring environment template or a high-availability (HA) authoring environment template.
 
 A single authoring environment contains two pods. One of the pods runs {CENTRAL}, the other runs {KIE_SERVER}. 
 ifdef::PAM[The {KIE_SERVER} by default includes an embedded H2 database engine.]
-This environment is most suitable for single-user authoring or when your OpenShift infrastructure has limited resources. 
+This environment is most suitable for single-user authoring or when your OpenShift infrastructure has limited resources. Also, it does not require persistent volumes that support the `ReadWriteMany` access mode.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/openshift/environment-authoring-con.adoc]: I would remove "Also,"

This environment is most suitable for single-user authoring or when your OpenShift infrastructure has limited resources. It does not require persistent volumes that support the `ReadWriteMany` access mode.

**Pull request merged**